packages/boxel-ui/addon/src/components/drag-and-drop/index.gts

```
@@ -0,0 +1,333 @@
+  @tracked hoveredColumnIndex: number | null = null;
+
+  constructor(owner: unknown, args: DndKanbanBoardArgs<Column>) {
+    super(owner, args);
+
+    if (!isFastBoot) {
+      this.loadModifiers();
+    }
+  }
+
+  async loadModifiers() {
+    try {
+      // @ts-expect-error Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node16', or 'nodenext'
+      const DndDraggableItemModifier = await import(
+        'ember-draggable-modifiers/modifiers/draggable-item'
+      );
+
+      // @ts-expect-error Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node16', or 'nodenext'
+      const DndDropTargetModifier = await import(
+        'ember-draggable-modifiers/modifiers/drop-target'
+      );
+
+      // @ts-expect-error Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node16', or 'nodenext'
+      const DndSortableItemModifier = await import(
+        'ember-draggable-modifiers/modifiers/sortable-item'
+      );
+      // @ts-expect-error Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node16', or 'nodenext'
+      const arrayUtils = await import('ember-draggable-modifiers/utils/array');
+
+      this.DndDraggableItemModifier = DndDraggableItemModifier.default;
+      this.DndDropTargetModifier = DndDropTargetModifier.default;
+      this.DndSortableItemModifier = DndSortableItemModifier.default;
+      this.insertAfter = arrayUtils.insertAfter;
+      this.insertAt = arrayUtils.insertAt;
+      this.insertBefore = arrayUtils.insertBefore;
+      this.removeItem = arrayUtils.removeItem;
+
+      this.isModifiersLoaded = true;
+    } catch (e) {
+      console.error('Error loading modifiers:', e);
```

why are we swallowing this error? shouldn't we let it explode?

---

also keep the try/catch scope as small as possible--only wrap the area that you expect to throw. there is no need to wrap all the assignments on 88-94 in a try/catch